kouncil-frontend/src/app/navbar/navbar.component.ts

```diff
@@ -1,21 +1,24 @@
-import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
-import { SearchService } from "app/search.service";
-import { Router } from "@angular/router";
+import {AfterViewInit, Component, ElementRef, OnInit, ViewChild} from '@angular/core';
+import {SearchService} from 'app/search.service';
+import {Router} from '@angular/router';
+import {Observable} from 'rxjs';
+import {HttpClient} from '@angular/common/http';
 
 @Component({
   selector: 'kafka-navbar',
   templateUrl: './navbar.component.html',
   styleUrls: ['./navbar.component.scss']
 })
-export class NavbarComponent implements OnInit {
+export class NavbarComponent implements OnInit, AfterViewInit {
```

Comment:
do czegoś używasz AfterViewInit?

Reply:
jest użyty do fucusa na szukajkę. ale to juz wcześniej było, w ramach tego PR poprawiłem tslinty

Reply:
👍